cli/cmd/compliance_aws.go

```diff
@@ -221,7 +225,172 @@ To show recommendation details and affected resources for a recommendation id:
 		},
 	}
 
-	// complianceAwsListAccountsCmd represents the search inside the aws command
+	// complianceAwsDisableReportCmd represents the disable-report sub-command inside the aws command
+	// experimental feature
+	complianceAwsDisableReportCmd = &cobra.Command{
+		Use:     "disable-report <report_type>",
+		Hidden:  true,
+		Aliases: []string{"disable"},
+		Short:   "Disable all recommendations for a given report type",
+		Long: `Disable all recommendations for a given report type.
+Supported report types are CIS_1_1
+
+To show the current status of recommendations in a report run:
+	lacework compliance aws status CIS_1_1
+
+To disable all recommendations for CIS_1_1 report run:
+	lacework compliance aws disable CIS_1_1
+`,
+		PreRunE: func(_ *cobra.Command, args []string) error {
+			switch args[0] {
+			case "CIS", "CIS_1_1", "AWS_CIS_S3":
```

Comment:
Same comment applies for the `get-report` cli command.  Is it nuts to code generate flags for every valid input?

How does a user know the valid inputs?  Instead we could have `disable-report --CIS_1_1 --AWS_CIS_S3` or something like that.

Reply:
That would be a nice to have, but going to leave as is due to the short lived nature of this command